client/me/concierge/shared/calendar-card.js

```diff
@@ -126,12 +126,12 @@ class CalendarCard extends Component {
 		const description = isDefaultLocale
 			? translate( 'Sessions are 30 minutes long.' )
 			: translate( 'Sessions are 30 minutes long and in %(defaultLanguage)s.', {
-					args: { defaultLanguage },
-				} );
+				args: { defaultLanguage },
+			} );
 
 		return (
 			<FoldableCard
-				className="concierge__calendar-card"
+				className="shared__calendar-card"
```

Comment:
Maybe just `calendar-card`? If the linter is requiring the underscores then maybe satisfy it with `concierge__calendar-card`

Reply:
Unfortunately the linter requires the parent folder prefix.